deps/v8/src/flag-definitions.h

```diff
@@ -178,7 +178,7 @@ DEFINE_BOOL(strong_this, true, "don't allow 'this' to escape from constructors")
 
 DEFINE_BOOL(es_staging, false, "enable all completed harmony features")
 DEFINE_BOOL(harmony, false, "enable all completed harmony features")
-DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony fetaures")
+DEFINE_BOOL(harmony_shipping, true, "enable all shipped harmony features")
```

Comment:
This comes from upstream. We may not be able to edit them directly like this, unless there is a compelling reason.